Automattic/sensei
# Render learner courses block with Sensei_Shortcode_User_Courses

Uses the current shortcode class to render the Learner Courses block. That also means using templates-based rendering to display the course list — `loop-courses.php` and `content-course.php`.

### Changes proposed in this Pull Request

* Use existing user courses shortcode class to render the block
* Add support to the shortcode class to disable elements based on the block settings

### Testing instructions
* Add `Learner Courses` block to a page & publish
* Visit page as a learner. Check that the learner courses are rendered.
* Check that editing the block to toggle elements applies on the frontend
--
* Check that the original My courses page using `[sensei_user_courses]` shortcode also still works
